subprojects/docs/src/docs/userguide/custom_gradle_types.adoc

```diff
@@ -102,6 +102,39 @@ In addition, the property must have one of the following types:
 - `MapProperty`
 - `ConfigurableFileCollection`
 
+[[sec:managed_nested_properties]]
+==== Read-only managed nested properties
+
+To declare a read-only managed nested property, add an abstract getter method for the property to the type annotated with `@link:{javadocPath}/org/gradle/api/tasks/Nested.html[Nested]`.
+The property should not have any setter methods.
+This pattern is useful the if current type has a nested complex type which has the same lifecycle.
+If the lifecycle is different, consider using `Property<NestedType>` instead.
```

Comment:
I like that you highlight the difference here, though I'm not sure "lifecycle" is defined well enough for this context. Not sure how to improve here, though.

Reply:
I could change it to talk about a one to one relationship between the current type and the nested complex type. WDYT?